### DISCUSSION TITLE
Start on collections

### DISCUSSION THREAD

**Discussion by doctordrogon** [ppy/osu#25781] [Ideas]
When I exit out of the main menu when I'm in a collection, and then go back in, im not still in the collection. Could the user be still on the collection even when they exit to the main menu and back?

**Comment by peppy** [comment]
You mean song select?

**Comment by doctordrogon** [comment]
Yea. The collections don’t have a sort of save state when you go from song
select to the main menu

**Comment by Wrongwayscout** [comment]
Found this when making my own discussion.

Collections should remain selected after returning from song select to main menu or multiplayer lobby.

I think this should behave like how stable does it where you don't have to reselect a collection every time you return to song select from somewhere other than a beatmap.
Most noticeable when selecting maps for a multiplayer queue, as every time you select a map and reenter song select to pick another it returns you to having no collection selected.
